packages/experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestionsList/FloatingSuggestionsList.styles.ts

```diff
@@ -30,7 +30,7 @@ export const getStyles = (props: IFloatingSuggestionsListStyleProps): IFloatingS
       classNames.suggestionsContainer,
       {
         overflowX: 'auto',
-        overflowY: 'hidden',
+        overflowY: 'auto',
```

Comment:
how does this interact with the headers/footers?

Reply:
![image](https://user-images.githubusercontent.com/64878908/94742735-0b1f2680-032b-11eb-82bf-ed741458c59a.png)
the header and footer are not affected,, this is only for the suggestions list